pyhoma/client.py

```diff
@@ -294,6 +295,8 @@ async def check_response(response: ClientResponse) -> None:
             result = await response.json(content_type=None)
         except JSONDecodeError:
             result = await response.text()
+            if "maintenance" in result:
```

Comment:
My preference would be to match on `Server is down for maintenance`? Just to be sure we don't catch an edge case where `maintenance` is somewhere in the report.